common.lua

```diff
@@ -73,6 +71,10 @@ beerchat.is_player_subscribed_to_channel = function(name, channel)
 		and (nil ~= beerchat.playersChannels[name][channel])
 end
 
+beerchat.sound_play = beerchat.enable_sounds and function (target, sound)
```

Comment:
smells like premature optimization here :smirk:

Reply:
It is more about moving check away from call site, intended to actually go inside that function later when actual per player and/or dynamic configuration is added. Currently it is hardcoded value in init.lua.

If you think syntax is bad idea I could change it right away and just leave check inside function? If you don't care then I'll leave it like it is for now and update when there's actually some way to configure it.